albumentations/augmentations/transforms.py

```diff
@@ -1622,6 +1622,15 @@ def __init__(
         self.quality_lower = quality_lower
         self.quality_upper = quality_upper
 
+    @classmethod
+    def prepare_init_args(cls, init_args):
+        if "compression_type" in init_args:
+            return {
+                **init_args,
+                "compression_type": getattr(ImageCompression.ImageCompressionType, init_args["compression_type"]),
+            }
+        return init_args
```

Comment:
Maybe would be better to inherit `ImageCompressionType` from `IntEnum` (currently it is inherited from `Enum`).
Then we can initialize `self.compression_type` like this `self.compression_type = int(compression_type)`

Reply:
@Dipet Changed the base class for ImageCompressionType to simplify serialization logic.